nengo/tests/test_processes.py

```diff
@@ -334,3 +334,108 @@ def test_present_input(Simulator, rng):
     y = sim.data[up].reshape(len(t), c, ni, nj)
     for k, [ii, image] in enumerate(zip(i, y)):
         assert np.allclose(image, images[ii], rtol=1e-4, atol=1e-7), (k, ii)
+
+
+class TestPiecewise(object):
+
+    dt = 0.001
```

Comment:
Unless there's some reason to change the `dt` in a specific test, it's probably best not to specify it, as specifying it indicates to the reader that we're going to be using a non-standard `dt` in some tests (which I don't believe we are here).